src/transformer.js

```diff
@@ -238,11 +287,13 @@ function _replaceMediaSources( xmlDoc, mediaMap ) {
     xmlDoc.find( '//*[@src] | //a[@href]' ).forEach( mediaEl => {
         const attribute = ( mediaEl.name().toLowerCase() === 'a' ) ? 'href' : 'src';
         const src = mediaEl.attr( attribute ).value();
-        const matches = src ? src.match( /jr:\/\/[\w-]+\/(.+)/ ) : null;
+        const matches = src ? src.match( /jr:\/\/[\s\w-]+\/(.+)/ ) : null;
         const filename = matches && matches.length ? matches[ 1 ] : null;
         const replacement = filename ? mediaMap[ filename ] : null;
         if ( replacement ) {
-            mediaEl.attr( attribute, replacement );
```

Comment:
The mediaMap used to be accessed and now it’s not anymore. What are the implications of that?

Reply:
It’s still using the `mediaMap` value (`replacement`), it’s just escaping before assignment. Or am I misunderstanding your question?

Reply:
🧠 not so good!

Reply:
No worries! Happens to me and my 🧠 all the time